src/DatadogLoggingService.js

```diff
@@ -47,7 +47,7 @@ class DatadogLoggingService extends NewRelicLoggingService {
       site: process.env.DATADOG_SITE,
       env: process.env.DATADOG_ENV,
       forwardErrorsToLogs: true,
-      sessionSampleRate: 100,
+      sessionSampleRate: parseInt(process.env.SESSION_SAMPLE_RATE || 0, 10),
```

Comment:
Do we want 100 as our default for now?

Reply:
i was thinking to default everything to 0 and then configure from envs. but if we want default values set, i'll update it

Reply:
should `sessionSampleRate` for both RUM and Browser logging differ?

i am currently using same env for both